hack/parallel.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -e
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2020 Red Hat, Inc.
+#
+
+NAMESPACE=${KUBEVIRT_INSTALLED_NAMESPACE:-kubevirt}
+TEMP_FILE=$(mktemp -p /tmp -t kubevirt.deploy.XXXX)
+
+trap 'rm -f $TEMP_FILE' EXIT ERR SIGINT SIGTERM SIGSTOP KILL
+
+function _kubectl() {
+    cluster-up/kubectl.sh "$@"
+}
+
+function clean_and_build() {
+    echo "Kubevirt namespace found, cleaning"
+    ./hack/cluster-clean.sh >$TEMP_FILE 2>&1 &
+    CLEAN_PID=$!
+
+    ./hack/cluster-build.sh
+
+    wait $CLEAN_PID
+    if [ $? -ne 0 ]; then
+        echo "Clean failed, output was:"
+        cat $TEMP_FILE
+        exit 1
+    fi
+}
+
+function build() {
+    echo "Kubevirt namespace not found, skipping clean"
+    ./hack/cluster-build.sh
+}
+
+function main() {
+    _kubectl get nodes >/dev/null 2>&1 && RC=$? || RC=$?
```

Comment:
You can just move the RC to the line below.

Reply:
we have -e , so if it doesnt find it it would fail without meaningful error and wont reach next line even, unless its fine by you

Reply:
done